jenkins/dockerized-e2e-runner.sh

```diff
@@ -30,7 +30,7 @@ mkdir -p "${HOST_ARTIFACTS_DIR}"
 : ${JENKINS_GCE_SSH_PRIVATE_KEY_FILE:='/var/lib/jenkins/gce_keys/google_compute_engine'}
 : ${JENKINS_GCE_SSH_PUBLIC_KEY_FILE:='/var/lib/jenkins/gce_keys/google_compute_engine.pub'}
 
-KUBEKINS_E2E_IMAGE_TAG='v20161004-f3120c3'
+KUBEKINS_E2E_IMAGE_TAG='v20161007-34648c8'
```

Comment:
Note that since we curl things in the image from other repos at head I don't think the hash is valuable except to prevent collisions when multiple images are released on the same day.